.github/workflows/black.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: psf/black@stable
         with:
           options: "--verbose"
-          version: "24.4.0"
```

Comment:
Why the change to black version here?  How does this relate to tenderly workflows?

Reply:
Figured this out in my next review... #908